packages/ddp-common/utils.js

```diff
@@ -38,80 +38,102 @@ export function last(array, n, guard) {
   return slice.call(array, Math.max(array.length - n, 0));
 }
 
-DDPCommon.SUPPORTED_DDP_VERSIONS = [ '1', 'pre2', 'pre1' ];
+DDPCommon.SUPPORTED_DDP_VERSIONS = [ '2', '1', 'pre2', 'pre1' ];
```

Comment:
I guess a new version of DDP would be necessary.

Things should be backwards compatible!